### DISCUSSION TITLE
Using Extent Report and I am not able to rename the Extent Report HTML File in Version 4? What is the problem

### DISCUSSION THREAD

**Discussion by allan10carlo86** [community/community#38259] [Other Feature Feedback, Questions, & Ideas]
Hi All, 

I have a code sample below: 

`public void setupOneTime()
        {
            string workingDirectory = Environment.CurrentDirectory;
            string projectDirectory = Directory.GetParent(workingDirectory).Parent.Parent.FullName;
            string reportPath = projectDirectory + "\\Extent.html";
            TestContext.WriteLine(reportPath);
            extentReports = new ExtentReports();
            var htmlReporter = new ExtentHtmlReporter(reportPath);
            extentReports.AttachReporter(htmlReporter);
            extentReports.AddSystemInfo("Host Name", "Local Host");
            extentReports.AddSystemInfo("Environment", "QA");
            extentReports.AddSystemInfo("User", "Allan");
            logger = LogManager.GetCurrentClassLogger();
            logger.Info(reportPath);
}
`

However, I don't see Extent.html in my folder instead I still see index.html

![image](https://user-images.githubusercontent.com/10335301/199939013-e69c980c-a431-4281-b605-80ba1cd14398.png)

I'm just wondering because I have already corrected my path and still I cannot rename the report.

Similar problem was posted also on this page: 

https://www.seleniumtraining.com/selenium-c-sharp-training/forum/extent-report-file-name-4652

I have not seen any fixes or workarounds on this. Can somebody help?

**Comment by ossanna16** [comment]
Hi there @allan10carlo86 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
